src/SFA.DAS.Reservations.Infrastructure/QueueNames.cs

```diff
@@ -10,5 +10,6 @@ public static class QueueNames
         public const string SignedAgreement = "SFA.DAS.Reservations.Jobs.SignedAgreement";
         public const string DraftApprenticeshipDeleted = "SFA.DAS.Reservations.Jobs.ApprenticeshipDeleted";
         public const string LevyAddedToAccount = "SFA.DAS.Reservations.Jobs.LevyAddedToAccount";
+        public const string UpdatedProviderPermissions = "SFA.DAS.ProviderRelationships.MessageHandlers";
```

Comment:
Can this be made to match the format of above so something more like `SFA.DAS.Reservations.Jobs.ProviderRelationshipsUpdated`